src/Microsoft.AspNetCore.Razor.Language/TagHelperSpanVisitor.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Language.Legacy;
+using Microsoft.AspNetCore.Razor.Language.Syntax;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal class TagHelperSpanVisitor : SyntaxRewriter
```

Comment:
>SyntaxRewriter [](start = 42, length = 14)

Does this need to be a rewriter? It doesn't change anything, it just produces output

Reply:
True. This can just be a SyntaxVisitor

---
In reply to: [233601788](https://github.com/aspnet/Razor/pull/2590#discussion_r233601788) [](ancestors = 233601788)